Core/CategoryDTO.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Core
+{
+    public class CategoryDTO
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename class 'CategoryDTO' to match pascal case naming rules, consider using 'CategoryDto'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4966361)